polybar-scripts/system-gpu-optimus/system-gpu-optimus.sh

```diff
@@ -1,53 +1,53 @@
 #!/bin/sh
 
-icon_intel="#1"
-icon_nvidia="#2"
-icon_hybrid="#3"
-
-hybrid_switching=0
+icon_intel="Intel"
+icon_nvidia="Nvidia"
+icon_hybrid="Hybrid"
 
+hybrid_switching=1
 
 gpu_current() {
-	mode=$(optimus-manager --print-mode)
-
-    echo "$mode" | cut -d ' ' -f 4
+  mode=$(optimus-manager --print-mode)
+  echo "$mode" | cut -d ' ' -f 5 | tr -d '[:space:]'
```

Comment:
This was originally returning a bunch of white space that caused the comparisons to all fail. I added `tr` to trim the white space.